### ISSUE TITLE
About get_info.py and the json file produced by it

### ISSUE THREAD

**Issue by TonyXu12138** [tonyhuang2022/UPL#5]
Hi there,
       Thanks for your great job! After running your code (the get_info.py). I noticed that the json file obtained will list all the training data with its id, confidence, logit and path. May I ask is that we must look through all the training data before we do the Top-K sampling?

**Comment by tonyhuang2022** [comment]
Looking more training data helps us to select better pseudo-labeled training images.

**Comment by TonyXu12138** [comment]
Thanks for replying!
May I ask an additional question that I noticed in your section4, you did the comparison with CoOp. However, the result shown in the table (CoOp's accuracy) is not exactly the same with what shown in CoOp's result document.( The difference is not huge). are those results derived by your own reproducing or by other method?

**Comment by tonyhuang2022** [comment]
Sorry for the late feedback.
If you run the CoOp's code multiple times, you will find that the performance of CoOp is not stable and it will fluctuate in a small range.
